docs/use-cases/image-data.md

```diff
@@ -110,26 +136,41 @@ From the **Select model** dropdown, select the name of your ML model service (fo
 {{% /tablestep %}}
 {{% tablestep %}}
 {{<imgproc src="/services/icons/modular-registry.svg" class="fill alignleft" style="max-width: 150px"  declaredimensions=true alt="Train models">}}
-**3. Configure the filtered camera**
+**4. Configure the filtered camera**
 
 The `filtered-camera` {{< glossary_tooltip term_id="modular-resource" text="modular component" >}} pulls the stream of images from the camera you configured earlier, and applies the vision service to it.
 
-Configure a `filtered-camera` component on your machine, following the [attribute guide in the README](https://github.com/erh/filtered_camera?tab=readme-ov-file#configure-your-filtered-camera) to specify the names of your webcam and vision service, and add classification and object detection filters.
+Configure a `filtered-camera` component on your machine, following the [attribute guide in the README](https://github.com/erh/filtered_camera?tab=readme-ov-file#configure-your-filtered-camera) to specify the names of your camera and vision service, and add classification and object detection filters.
 
 {{% /tablestep %}}
 {{% tablestep %}}
 {{<imgproc src="/services/icons/data-capture.svg" class="fill alignleft" style="max-width: 150px"  declaredimensions=true alt="Train models">}}
-**4. Configure data capture and sync on the filtered camera**
+**5. Configure data capture and sync on the filtered camera**
 
-Configure data capture and sync just as you would for a webcam.
+Configure data capture and sync just as you did before.
 The filtered camera will only capture image data that passes the filters you configured in the previous step.
 
-Turn off data capture on your webcam if you haven't already, so that you don't capture duplicate or unfiltered images.
+Turn off data capture on your camera if you haven't already, so that you don't capture duplicate or unfiltered images.
+
+{{% /tablestep %}}
+{{% tablestep %}}
+
+{{<imgproc src="/services/ml/collect.svg" class="fill alignleft" style="max-width: 150px"  declaredimensions=true alt="Train models">}}
+**6. View filtered data in the Viam app**
+
+Once you save your configuration, your filtered images will sync and you can [view those images in the Viam app](/services/data/view/) from the **DATA** tab.
+
+{{< alert title="Wait until data appears" color="note" >}}
+Your data will sync at the specified sync interval, which may mean you have to wait for data to appear.
+
+If no data appears after the sync interval, check the [**Logs**](/cloud/machines/#logs) and ensure that the condition for filtering is met.
+You can add a [`transform` camera](/components/camera/transform/) to see detections or classifications live from the [**CONTROL** tab](/cloud/machines/#control).
```

Comment:
This seems slightly unrelated to this alert. Consider moving/removing or at least adding a new line to make a separate paragraph.

Reply:
removed the alert around it